express-post-json/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res, next) => {
+  const gradesArr = [];
+  for (const property in grades) {
+    gradesArr.push(grades[property]);
+  }
+  res.json(gradesArr);
+  next();
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  if (req.method !== 'POST') {
+    res.status(400).send('bad request');
+    return;
+  }
```

Comment:
Since you are using `app.post` only a `post` request will ever reach this callback function so there is no need to check the request method.